examples/individual_task_with_parameters/databricks.yml

```
@@ -0,0 +1,25 @@
+default: # will be applied to all workflows
+  job_clusters:
+    - job_cluster_key: default
+      new_cluster:
+        spark_version: 7.3.x-scala2.12
+        node_type_id: Standard_DS3_v2
+        num_workers: 2
+        spark_env_vars:
+          KEDRO_LOGGING_CONFIG: /dbfs/FileStore/<package-name>/conf/logging.yml
+    - job_cluster_key: high-performance
+      new_cluster:
+        spark_version: 7.3.x-scala2.12
+        node_type_id: Standard_DS3_v2
+        num_workers: 8
+        spark_env_vars:
+          KEDRO_LOGGING_CONFIG: /dbfs/FileStore/<package-name>/conf/logging.yml
+  tasks: # will be applied to all tasks in each workflow
+    - task_key: default
+      job_cluster_key: default
+    - task_key: make_predictions
+      job_cluster_key: high-performance
+      python_wheel_task:
+        parameters:
+          - --load-version
+          - "2025-06-12"
```

```suggestion
          - --load-versions
          - "dataset:2025-06-12"
```

To match with the kedro interface:

https://docs.kedro.org/en/stable/data/data_catalog.html#dataset-versioning